README.md

```diff
@@ -38,3 +38,13 @@ The event is received by the repository which triggers workflows (simulate what
 ## Extra
 
 - [Dependabot configuration](.github/dependabot.yml): as configured in repository using dependabot to update dependencies
+
+
+## Test to update the readme of https://github.com/process-analytics/bpmn-visualization-R
```

Comment:
I suggest we found a better wording that explain that the following paragraph is use to mimic the R package README. 
It was not obvious to me at a first glance that the content was added for this purpose